walkthroughs/howto-ingress-gateway/infrastructure/ecs-service.yaml

```diff
@@ -552,7 +552,6 @@ Resources:
         - Name: 'app'
```

Comment:
Am I missing something or does this task definition not a have a ProxyConfiguration?

Reply:
Oh I actually understand now that its not necessary since the proxy isn't running sidecar. And that explains the uid change

Reply:
It does not.

AFAICT it doesn't need a ProxyConfiguration since Envoy is running as a gateway.